OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs

```diff
@@ -337,16 +413,21 @@ private async void MediaPlayer_TimeChanged(object sender, EventArgs e)
                 // when the user is not seeking.
                 if (!IsSeeking)
                 {
-                    // Sometimes the MediaPlayer is already null upon
+                    // Sometimes the _mediaPlayer is already null upon
                     // navigating away and this still gets called.
-                    if (MediaPlayer != null)
+                    if (_mediaPlayer != null)
                     {
-                        TimeLineValue = MediaPlayer.Time;
+                        TimeLineValue = _mediaPlayer.Time;
                     }
                 }
             });
         }
 
+        private void MediaPlayer_MediaChanged(object sender, EventArgs e)
+        {
+            _isFirstPlaying = true;
```

Comment:
this is called on a reload aswell, is that intended?

Reply:
Good catch. I added this one for the previous/next episode functionality to also set this boolean. It was not intended to change on reload.